aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie50/SnapShots/Edit.cshtml.cs

```diff
@@ -0,0 +1,76 @@
+﻿#if Never
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using RazorPagesMovie.Models;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RazorPagesMovie.Pages.Movies
+{
+#region snippet2
+    public class EditModel : PageModel
+    {
+        private readonly RazorPagesMovie.Data.RazorPagesMovieContext _context;
+
+        public EditModel(RazorPagesMovie.Data.RazorPagesMovieContext context)
+        {
+            _context = context;
+        }
+
+        [BindProperty]
+        public Movie Movie { get; set; }
+
+        public async Task<IActionResult> OnGetAsync(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            Movie = await _context.Movie.FirstOrDefaultAsync(m => m.ID == id);
```

Comment:
I didn't confirm that this compiles, but see if you can simplify to the following:

```suggestion
            Movie = await _context.Movie.FindAsync(id);
```

Reply:
@scottaddie, It compiles and runs, however, that code is what the scaffolding generates, and we want the user in this case to use what the scaffolding creates.

So I will not make the change.  However, I think you have a good argument for a small improvement in the scaffolding code of the product.  :)